src/qt/qvalidatedlineedit.cpp

```diff
@@ -106,6 +106,7 @@ void QValidatedLineEdit::checkValidity()
 void QValidatedLineEdit::setCheckValidator(const QValidator *v)
 {
     checkValidator = v;
+    checkValidity();
```

Comment:
b1a544be109d336c0b53722e3f8b51687972c94e

What bug is this change fixing? Or is this just making the code more correct?

Reply:
bump

this is also not evident to me

Reply:
> Re-check validity of input when we first set the validator (probably a no-op in practice).

Reply:
Maybe add "Probably a no-op in practice." to the commit message.